app/javascript/src/Navigation/components/VerticalNav.jsx

```diff
@@ -1,14 +1,19 @@
 // @flow
 
 import React from 'react'
-import { Nav, NavExpandable, NavItem, NavList, NavGroup } from 'Navigation/components/PF4NavProxy'
+import {
+  Nav,
+  NavExpandable,
+  NavGroup,
+  NavItem,
+  NavList
+} from '@patternfly/react-core'
 import { createReactWrapper } from 'utilities/createReactWrapper'
 
 type Item = {
   id: string,
   title: string,
-  path: ?string,
-  target: ?string
```

Comment:
What was the `target` used for before?

Reply:
It was passed to `NavItem` in `PF4NavProxy.jsx`. Several props were passed because the components were crafted by us. But as we are using now the PF4 components, we don't need it anymore

Reply:
This target is used for some links that are opened in a new tab. Therefore by removing it, it won't work. In other words, it can't be removed.